sos/plugins/openstack_cinder.py

```diff
@@ -45,6 +47,9 @@ def setup(self):
         else:
             self.add_copy_spec_limit("/var/log/cinder/*.log",
                                      sizelimit=self.limit)
+        if self.get_option("exclude"):
+            exclude = self.get_option("exclude")
+            self.add_forbidden_path(exclude)
```

Comment:
If we're taking this route, then just exclude the files and be done with it - adding an `exclude` option, then forbidding things if it is set, and *then* going on to collect them anyway is just confusing for everyone involved.

Also, what's the point of setting the `exclude` variable here, since it's not used anywhere?